python_benchmarks/global_params.py

```diff
@@ -2,6 +2,41 @@
 from typing import Union, List, Tuple
 from nvfuser import DataType
 from .core import DEVICE_PROPERTIES
+import numpy as np
+
+# Model Parameters from LLMs (GPT2/3, PaLM, LLama)
+
+# Input sequence length
+SEQ_LENGTH_MIN = 1024
+SEQ_LENGTH_MAX = 16384
```

Comment:
Are they used anywhere?

Reply:
No, they are not. I was hoping to use them in attention benchmarks.
I'll remove them for now and add as needed.

Reply:
(Same for other `*_MIN` and `*_MAX` variables)